pxteditor/editor.ts

```diff
@@ -171,6 +172,9 @@ namespace pxt.editor {
         openInstructions(): void;
         closeFlyout(): void;
 
+        showBanner(): void;
```

Comment:
Could this be one function accepting a bool. Just to reduce the number of functions specific to this component in apptsx